AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/PermissionsManager.cs

```diff
@@ -0,0 +1,75 @@
+// The PermissionsManager class implements all the methods needed to 
+// check whether the user has the necessary permissions to access a settingGroup
+// and allows the user to request permission to a settingGroup if already don't have it
+// The user (owner of the project or admin) can then approve or reject the request
+public class PermissionsManager
+{
+    private readonly List<Permission> _permissions;
+
+    public PermissionsManager()
+    {
+        _permissions = new List<Permission>();
+    }
```

Comment:
```suggestion
    private readonly SettingsContext _settingsContext;
    private readonly IBidirectionalConverter<Model.Permission, Permission> _permissionConverter;

    public MySqlPermissionsRepository(SettingsContext settingsContext, IBidirectionalConverter<Model.Permission, Permission> permissionConverter)
    {
        _settingsContext = settingsContext;
        _permissionConverter = permissionConverter;
    }
```

This should pretty much match the other repositories that we already have